### PR TITLE
Updating phone numbers to have the 1-.

### PR DESCRIPTION
**Pull request by robbiethegeek-usds**

### CODE REVIEW

File: content/pages/health-care/about-va-health-care.md [hunk 1]
```diff
@@ -225,7 +225,7 @@ If you have other forms of health care coverage (like a private insurance plan,
 <div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
```
Comment (phiden) [inline content/pages/health-care/about-va-health-care.md]: Should there be a +1 before 877-222-VETS?

File: content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md [hunk 2]
```diff
@@ -65,7 +65,7 @@ You may be able to use one or more of these services if:
 
 We may also consider other factors, like your service-connected disability status or insurance coverage.
 
-To find out how to access these services, contact your VA social worker. Or, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). 
+To find out how to access these services, contact your VA social worker. Or, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
```
Comment (phiden) [inline content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md]: Should there be a +1 before 877-222-VETS?

File: content/pages/health-care/apply.md [hunk 3]
```diff
@@ -49,7 +49,7 @@ Once you’ve figured out if you qualify, applying for VA health care benefits i
 <div itemprop="acceptedAnswer" itemscope itemtype="http://schema.org/Answer">
 <div itemprop="text">
 
-Call our toll-free hotline at 877-222-VETS (<a href="tel:+1phonenumber">877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). 
+Call our toll-free hotline at 877-222-VETS (<a href="tel:+1phonenumber">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
```
Comment (phiden) [inline content/pages/health-care/apply.md]: Should there be a +1 before 877-222-VETS?

File: content/pages/health-care/eligibility.md [hunk 4]
```diff
@@ -68,7 +68,7 @@ If none of the above apply to you, you may still qualify for care based on your
 
 #### What if I need help filling out my application?
 
-Call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
+Call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET).
```
Comment (phiden) [inline content/pages/health-care/eligibility.md]: Should there be a +1 before 877-222-VETS?

File: content/pages/health-care/health-conditions/exposure-to-hazardous-materials.md [hunk 5]
```diff
@@ -157,4 +157,4 @@ Chemicals, paints, radiation, and other hazards you may have come in contact wit
 
 <br>
 
-If you need help accessing services, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). Or, [contact your state’s Veterans agency](https://www.va.gov/statedva.htm).
+If you need help accessing services, call our toll-free hotline at 877-222-VETS (<a href="tel:+1-877-222-8387">1-877-222-8387</a>), Monday through Friday, 8:00 a.m. to 8:00 p.m. (ET). Or, [contact your state’s Veterans agency](https://www.va.gov/statedva.htm).
```
Comment (phiden) [inline content/pages/health-care/health-conditions/exposure-to-hazardous-materials.md]: Should there be a +1 before 877-222-VETS?

File: content/pages/health-care/health-conditions/mental-health.md [hunk 6]
```diff
@@ -107,10 +107,10 @@ You may still be able to get care:
 - **If you’ve served in a combat zone,** get free private counseling, alcohol and drug assessment, and other support at one of our 300 community Vet Centers. [Find a Vet Center near you](/facilities/). 
 - **If you’re homeless or at risk of becoming homeless:** 
   - Visit our website to learn about VA programs for Veterans who are homeless. [Learn about our homelessness programs](https://www.va.gov/homeless/).
-  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
+  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">1-877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
```
Comment (phiden) [inline content/pages/health-care/health-conditions/mental-health.md]: Should there be a +1 before 877-4AID-VET?

File: content/pages/health-care/health-conditions/mental-health/depression.md [hunk 7]
```diff
@@ -81,7 +81,7 @@ You may still be able to get care:
 - **If you’ve served in a combat zone,** get free private counseling, alcohol and drug assessment, and other support at one of our 300 community Vet Centers. [Find a Vet Center near you](/facilities/). 
 - **If you’re homeless or at risk of becoming homeless:** 
   - Visit our website to learn about VA programs for Veterans who are homeless. [Learn about our homelessness programs](https://www.va.gov/homeless/).
-  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential (private).
+  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">1-877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential (private).
```
Comment (phiden) [inline content/pages/health-care/health-conditions/mental-health/depression.md]: Should there be a +1 before 877-4AID-VET?

File: content/pages/health-care/health-conditions/military-sexual-trauma.md [hunk 8]
```diff
@@ -78,10 +78,10 @@ Here’s how to access VA services for MST:
 - **If you’ve served in a combat zone,** get free private counseling, alcohol and drug assessment, and other support at one of our 300 community Vet Centers. [Find a Vet Center near you](/facilities/). 
 - **If you’re homeless or at risk of becoming homeless:** 
   - Visit our website to learn about VA programs for Veterans who are homeless. [Learn about our homelessness programs](https://www.va.gov/homeless/).
-  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
+  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">1-877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
```
Comment (phiden) [inline content/pages/health-care/health-conditions/military-sexual-trauma.md]: Should there be a +1 before 877-4AID-VET?

File: content/pages/health-care/health-conditions/substance-use-problems.md [hunk 9]
```diff
@@ -104,7 +104,7 @@ You may still be able to get care:
 - **If you’ve served in a combat zone,** get free private counseling, alcohol and drug assessment, and other support at one of our 300 community Vet Centers. [Find a Vet Center near you](/facilities/). 
 - **If you’re homeless or at risk of becoming homeless:** 
   - Visit our website to learn about VA programs for Veterans who are homeless. [Learn about our homelessness programs](https://www.va.gov/homeless/).
-  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential (private).
+  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">1-877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential (private).
```
Comment (phiden) [inline content/pages/health-care/health-conditions/substance-use-problems.md]: Should there be a +1 before 877-4AID-VET?

File: content/pages/health-care/health-conditions/womens-health-care-needs.md [hunk 10]
```diff
@@ -88,19 +88,19 @@ Once you’ve signed up:
 - Talk to your primary care provider, who will connect you with any services you may need, **or**
 - Contact the Women Veterans Program Manager at your local VA medical center. She will help coordinate all the services you may need—from primary care to specialized care for chronic (long-lasting) conditions or reproductive health. To connect with your local Women Veterans Program Manager:
   - Contact your local medical center and ask for the Women Veterans Program Manager ([find a VA medical center near you](/facilities/)), **or** 
-  - Call our Women Veterans Call Center at 855-VA-WOMEN (<a href="tel:+1-855-829-6636">855-829-6636</a>) and we’ll connect you with your local manager.
+  - Call our Women Veterans Call Center at 855-VA-WOMEN (<a href="tel:+1-855-829-6636">1-855-829-6636</a>) and we’ll connect you with your local manager.
```
Comment (phiden) [inline content/pages/health-care/health-conditions/womens-health-care-needs.md]: Should there be a +1 before 855-VA-WOMEN?

File: content/pages/health-care/health-conditions/womens-health-care-needs.md [hunk 11]
```diff
@@ -88,19 +88,19 @@ Once you’ve signed up:
...
 - **If you’ve served in a combat zone,** get free private counseling, alcohol and drug assessment, and other support at one of our 300 community Vet Centers. [Find a Vet Center near you](/facilities/). 
 - **If you’re homeless or at risk of becoming homeless:** 
   - Visit our website to learn about VA programs for Veterans who are homeless. [Learn about our homelessness programs](https://www.va.gov/homeless/).
-  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
+  - Contact the National Call Center for Homeless Veterans at 877-4AID-VET (<a href="tel:+1-877-424-3838">1-877-424-3838</a>) anytime day or night. A trained VA counselor will offer information about VA homeless programs, health care, and other services in your area. The call is free and confidential.
```
Comment (phiden) [inline content/pages/health-care/health-conditions/womens-health-care-needs.md]: Should there be a +1 before 855-4AID-VET?

File: content/pages/health-care/health-conditions/womens-health-care-needs.md [hunk 12]
```diff
@@ -88,19 +88,19 @@ Once you’ve signed up:
...
+  - Call our Women Veterans Call Center at 855-VA-WOMEN (<a href="tel:+1-855-829-6636">1-855-829-6636</a>) and we’ll connect you with your local manager.
 
 <br>
 
 ### What if I don’t have VA health care benefits? 
 
 If you don’t have VA health care benefits, you may still be able to get care:
 
-- **If you’re not sure what health benefits you may have earned,** contact our Women Veterans Call Center. Our trained, all-female staff will help you figure out your benefits and take the next step to getting care. Call 855-VA-WOMEN (<a href="tel:+1-855-829-6636">855-829-6636</a>). Or, [start a secure chat](https://www.womenshealth.va.gov/WOMENSHEALTH/programoverview/wvcc.asp).  
+- **If you’re not sure what health benefits you may have earned,** contact our Women Veterans Call Center. Our trained, all-female staff will help you figure out your benefits and take the next step to getting care. Call 855-VA-WOMEN (<a href="tel:+1-855-829-6636">1-855-829-6636</a>). Or, [start a secure chat](https://www.womenshealth.va.gov/WOMENSHEALTH/programoverview/wvcc.asp).
```
Comment (phiden) [inline content/pages/health-care/health-conditions/womens-health-care-needs.md]: Should there be a +1 before 855-VA-WOMEN?

File: content/pages/health-care/health-conditions/womens-health-care-needs.md [hunk 13]
```diff
@@ -115,4 +115,4 @@ If you’re a Veteran who has experienced MST, you can get help through VA. You
 
 - Our women’s health website gives you the latest information about how we help women Veterans stay well. [Visit the women’s health website](https://www.womenshealth.va.gov/).
 - Our Center for Women Veterans website tells you about health care and other benefits and programs for women Veterans. [Visit the Center for Women Veterans website](https://www.va.gov/womenvet/). 
-- Our Women Veterans Call Center representatives can help you understand your VA benefits and find the right services to fit your needs. Get free, confidential guidance. Call the Center at 855-VA-WOMEN (<a href="tel:+1-855-829-6636">855-829-6636</a>), Monday through Friday, 8:00 a.m. to 10:00 p.m. (ET) and Saturday, 8:00 a.m. to 6:30 p.m. (ET).
+- Our Women Veterans Call Center representatives can help you understand your VA benefits and find the right services to fit your needs. Get free, confidential guidance. Call the Center at 855-VA-WOMEN (<a href="tel:+1-855-829-6636">1-855-829-6636</a>), Monday through Friday, 8:00 a.m. to 10:00 p.m. (ET) and Saturday, 8:00 a.m. to 6:30 p.m. (ET).
```
Comment (phiden) [inline content/pages/health-care/health-conditions/womens-health-care-needs.md]: Should there be a +1 before 855-VA-WOMEN?

File: content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md (hunk 2 above)
Comment (robbiethegeek-usds) [reply content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md]: @bethpotts ^^

Comment (bethpotts) [reply content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md]: Yes, please!

Comment (bethpotts) [reply content/pages/health-care/about-va-health-care/assisted-living-and-home-health-care.md]: For all the examples Sophia found of vanity phone numbers, please do include the 1-

**Comment by robbiethegeek-usds** [comment]
All those vanity phone numbers should be updated

File: content/pages/health-care/about-va-health-care.md (hunk 1 above)
Comment (robbiethegeek-usds) [reply content/pages/health-care/about-va-health-care.md]: Yup updated.

**Review by phiden — APPROVED** [review]

**Pull request merged**